sigh/lib/assets/resign.sh

```diff
@@ -380,6 +380,30 @@ function add_provision {
     add_provision_for_bundle_id "$PROVISION" "$BUNDLE_ID"
 }
 
+# returns value from plist
+# plist_vale $PLIST $KEY
+function plist_value {
+  local PLIST=$1
+  local KEY=$2
+
+  echo "$(PlistBuddy -c "Print :$KEY" "$PLIST")"
+}
+
+# changes key in plist
+# change_plist_key $PLIST $KEY $VALUE
+function change_plist_value {
+  local PLIST=$1
+  local KEY=$2
+  local VALUE=$3
+  
+  if [[ -n "$VALUE" ]];
+  then
+      CURRENT_VALUE="$(plist_value "$PLIST" $KEY)"
+      log "Updating the $KEY from '$CURRENT_VALUE' to '$VALUE'"
+      PlistBuddy -c "Set :$KEY $VALUE" "$PLIST"
```

Comment:
What do you think of *not* setting the `$VALUE` if it is equal to `$CURRENT_VALUE`? Saves a small bit of time.